backend/src/models/item-by-tag.model.ts

```diff
@@ -0,0 +1,14 @@
+import { prop } from '@typegoose/typegoose';
+import { Expose } from 'class-transformer';
+import { Item } from './item.model';
+import { Tag } from './tag.model';
+
+export class ItemsByTag {
+  @Expose()
+  @prop({ required: true, unique: true, type: () => Tag })
```

Comment:
Die Datenstruktur versteh ich nicht ganz, aber das ist wahrscheinlich eh teil von wip kram

Reply:
Ja work in progress, da muessen wir gucken wie wir es machen dass die Search optimiert ist, vermutlich sogar den gesamten Tag aufdroeseln als dass `ItemsByTag extends Tag` ist